dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass AppNotificationDevicesData
+    {
+        // Initializes a new Instance of NotificationDevicesData
+        AppNotificationDevicesData();
+
+        // Checks if Video or Audio Calling is supported
+        static Boolean IsVideoOrAudioCallingSupported();
+
+        // Gets or sets the Camera Device Id
+        String CameraDeviceId{ get; set; };
```

Comment:
The Media stack uses the terms "VideoDevice" and "AudioDevice" - rename these to:

* VideoDeviceId
* AudioInputDeviceId
* AudioOutputDeviceId

(The user might be using a synthetic audio input/ouput stack and not a physical camera / mic / speaker).